.github/PULL_REQUEST_TEMPLATE/general_change.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
+<!-- 
+Please include a summary of the change. Please also include relevant motivation and context.
+-->
+
+Fixes #(issue)
+
+<!--
+Note: You can remove the "Fixes #(issue)" if you don't plan on making this PR close an issue.
+-->
+
+## Type of change
+<!-- Replace :x: with :white_check_mark: to "check" the specified bullet -->
+
+:x: Bug fix (non-breaking change which fixes an issue)
```

Comment:
Is there a reason for using `:x:` instead of regular markdown checkbox `[ ]`? I think that would be easier to understand for users.

Reply:
It's what we were doing already
Also if you use checkboxes to represent multiple choice, they get interpreted as tasks by GitHub

Reply:
Precisely. We used to have check boxes in the Lawnchair repo https://github.com/LawnchairLauncher/lawnchair/commit/e7e56eff483f298381b3b20de5664b6bcbebbc72, but it was displaying the unwanted task list, so we changed it. I've basically copied the template from Lawnchair and modified it a little.

Reply:
Gotcha!